fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractPersistenceTest.java

```diff
@@ -97,6 +101,11 @@ public void commitTrx() throws Exception{
             persistence.getTransaction().commit();
         }
     }
+    
+    @AfterClass(alwaysRun = true)
```

Comment:
Look at AfterSuite

Reply:
is it ok to have the same AfterSuite method on every test class?